src/vm/gccover.cpp

```diff
@@ -940,6 +943,34 @@ bool replaceInterruptibleRangesWithGcStressInstr (UINT32 startOffset, UINT32 sto
 }
 #endif
 
+// Is this a call instruction to JIT_RareDisableHelper()
+bool isGCStopHelper(SLOT instrPtr)
+{
+#if defined(_TARGET_ARM64_)
+   if (((*reinterpret_cast<DWORD*>(instrPtr)) & 0xFC000000) == 0x94000000)
+   {
+       // call through immediate
+       int imm26 = ((*((DWORD*)instrPtr)) & 0x03FFFFFF)<<2;
+       // SignExtend the immediate value.
+       imm26 = (imm26 << 4) >> 4;
+       DWORD* target = (DWORD*) (instrPtr + imm26);
+       // Call to jithelpers happens via jumpstub
+       if(*target == 0x58000050 /* ldr xip0, PC+8*/ && *(target+1) == 0xd61f0200 /* br xip0 */)
+       {
+           // get the actual jithelper target
+           target = *(((DWORD**)target) + 1);
+           if((TADDR)target == GetEEFuncEntryPoint(JIT_RareDisableHelper))
+           {
+               return true;
+           }
+       }
+   }
+   return false;
+#elif defined(_TARGET_ARM_)
+   // TODO
+#endif    
```

Comment:
Thanks. Will fix this. Currently I only care about arm64 gcstress run results. Still need to add changes for arm.